test/spec/modules/stroeerCoreBidAdapter_spec.js

```diff
@@ -796,12 +797,37 @@ describe('stroeerCore bid adapter', function() {
             }
           }
         });
+
+        it('config key values should override identical metatag key values', () => {
```

Comment:
I would rename since we are testing more than one use case:

```suggestion
        it('should merge key values with config key values taking precedence', () => {
```